src/components/PatientsTable/PatientsTable.test.tsx

```diff
@@ -0,0 +1,39 @@
+//
+// This source file is part of the Stanford Biodesign Digital Health Spezi Web Health Components open-source project
+//
+// SPDX-FileCopyrightText: 2024 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import { type Patient } from '@medplum/fhirtypes'
+import { render, screen } from '@testing-library/react'
+import { PatientsTable } from '.'
+
+describe('PatientsTable', () => {
+  it('renders element', () => {
+    const onDelete = vitest.fn()
+    const editRoute = '/patients/edit'
+    const data: Patient[] = [
+      { id: '1', name: [{ given: ['Leland'], family: 'Stanford' }] } as Patient,
+    ]
```

Comment:
`Patient[]` is unnecessary. TypeScript already knows it's `Patient[]` due to `as Patient` declaration.